tests/foreman/ui/test_repository.py

```diff
@@ -296,8 +291,7 @@ def test_positive_update_GPG(self, repo_name):
                 repo_name, 'gpgkey', gpgkey_2.name))
 
     @run_only_on('sat')
-    @data(*generate_strings_list())
-    def test_positive_update_checksum_type(self, repo_name):
+    def test_positive_update_checksum_type(self):
```

Comment:
left test_positive_update_URL as datadriven and changed others to normal tests to avoid test redundancy.